src/index.html

```diff
@@ -127,6 +128,67 @@ <h2 class="chart-period mb text-center">
           </small>
         </p>
       </footer>
+
+      <div id="settings-panel" class="settings hidden">
+        <button id="close-settings" class="close-button">&times;</button>
+        <h2>Customize Panel</h2>
+        <div class="settings-container">
+          <label>Theme:</label>
+          <div class="toggle-container">
+            <div id="theme-toggle" class="toggle">
+              <div class="toggle-option active" data-theme="light">Light</div>
+              <div class="toggle-option" data-theme="dark">Dark</div>
+            </div>
```

Comment:
There must be a 3rd option - system default. Is there a way for us to detect the system default one?

Would be very useful especially for operating systems that have scheduled light / dark mode based on the time of the day.

Reply:
I’ve implemented the system theme functionality. The application now detects the system's default theme (light or dark) and applies it accordingly. Let me know if you encounter any issues.